rmf_building_sim_common/CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+## Changelog for package rmf\_building\_sim\_common
+
```

Comment:
We need a section for 2.0.0 and the related entries.

Reply:
Done https://github.com/open-rmf/rmf_simulation/pull/90/commits/6dce25ffbc3f2581965b9052824ad5e39a3e9f9f